smac/optimizer/default_priors.py

```diff
@@ -0,0 +1,53 @@
+
```

Comment:
same here. Do we need all of them?

Reply:
And also same for my previous suggestions.